Zotero/Scenes/Detail/PDF/PDFCoordinator.swift

```diff
@@ -36,6 +36,7 @@ protocol PdfReaderCoordinatorDelegate: AnyObject {
     func showFontSizePicker(sender: UIView, picked: @escaping (UInt) -> Void)
     func showDeleteAlertForAnnotation(sender: UIView, delete: @escaping () -> Void)
     func showTagPicker(libraryId: LibraryIdentifier, selected: Set<String>, userInterfaceStyle: UIUserInterfaceStyle?, picked: @escaping ([Tag]) -> Void)
+    func showDocumentChangedAlert(key: String, parentKey: String?, libraryId: LibraryIdentifier, completed: @escaping () -> Void)
```

Comment:
The first parameters seem not to be used at all.

Reply:
Removed in 9b5ebbbe2e66e8aa4763a6155777bfb0053b416e